Open Judge System/Web/OJS.Web/Controllers/TempController.cs

```diff
@@ -41,5 +47,42 @@ public ActionResult RegisterJobForDeletingLeftOverFilesInTempFolder()
 
             return null;
         }
+
+        public ActionResult NormalizeParticipants()
```

Comment:
`NormalizeParticipantsWithDuplicatedParticipantScores` would be a more descriptive name